v0.5/recommendation/README.md

```diff
@@ -0,0 +1,262 @@
+# MLPerf Inference Benchmarks for Recommendation Task
+
+This is the reference implementation for MLPerf Inference benchmarks.
+
+### Supported Models
+
+| model | framework | accuracy | AUC | dataset | trained  | precision | notes |
+| ---- | ---- | ---- | ---- | ---- | ---- | ---- | ---- |
+| dlrm | PyTorch | 78.9% | N/A | [Criteo Kaggle DAC](https://labs.criteo.com/2014/02/kaggle-display-advertising-challenge-dataset/)       | N/A                                                                     | fp32 |                          |
+| dlrm | PyTorch | 81.07% | N/A | [Criteo Terabyte](https://labs.criteo.com/2013/12/download-terabyte-click-logs/) (0.875) | [weights](https://dlrm.s3-us-west-1.amazonaws.com/models/tb0875_10M.pt) | fp32 | --max-ind-range=10000000 --data-sub-sample-rate=0.875 |
+| dlrm | PyTorch | N/A | 80.25% | [Criteo Terabyte](https://labs.criteo.com/2013/12/download-terabyte-click-logs/)         | [weights](https://dlrm.s3-us-west-1.amazonaws.com/models/tb00_40M.pt)   | fp32 | --max-ind-range=40000000 |
+
+### Disclaimer
+This benchmark app is a reference implementation that is not meant to be the fastest implementation possible.
+It is written in python which might make it less suitable for large number of CPU's.
+
+The reference implementation includes all required pre-processing of datasets.
+It also includes a ```--accuracy``` option to validate accuracy and AUC as required by mlperf.
+If you are not using the reference implementation, a few scripts will help:
+
+## Prerequisites and Installation
+We support [PyTorch](http://pytorch.org) and might add TensorFlow backend implementation.
+
+The following steps are **only** needed if you run the benchmark **without Docker**.
+
+Python 3.5, 3.6 or 3.7 is supported and we recommend to use Anaconda (See [Dockerfile](Dockerfile.cpu) for a minimal Anaconda install).
+
+Install the desired backend. For pytoch:
+```
+pip install torch torchvision
+pip install scikit-learn
+pip install numpy
+pip install pydot
+pip install torchviz
+pip install protobuf
+pip install tqdm
+```
+
+### Prepare the code and dataset
+1. Download or clone the MLPerf [inference](https://github.com/mlperf/inference) and [trainining](https://github.com/mlperf/training) code
+```
+cd $HOME
+mkdir ./mlperf && cd ./mlperf
+git clone --recurse-submodules https://github.com/mlperf/training.git
+git clone --recurse-submodules https://github.com/mlperf/inference.git
+export DLRM_DIR=$HOME/mlperf/training/recommendation
+```
+2. Download pre-trained model weights
+```
+cd $HOME/mlperf/inference/v0.5/recommendation
+mkdir ./model && cd ./model
+mv <downloaded_file> dlrm_terabyte.pytorch
+export MODEL_DIR=./model
+```
+3. Download Criteo dataset
+```
+cd $HOME/mlperf/inference/v0.5/recommendation
+mkdir ./criteo && cd ./criteo
+mv <downloaded_file> ./
```

Comment:
Is it just one file or several?

Reply:
It' multiple files, which should be downloaded and placed into the same directory. They are described in more details in the dataset section.

Reply:
@googlebot I consent